cmd/gossamer/genesis.go

```diff
@@ -4,11 +4,19 @@ import (
 	"github.com/ChainSafe/gossamer/cmd/utils"
 	cfg "github.com/ChainSafe/gossamer/config"
 	"github.com/ChainSafe/gossamer/config/genesis"
+	"github.com/ChainSafe/gossamer/polkadb"
+	"github.com/ChainSafe/gossamer/trie"
 	log "github.com/ChainSafe/log15"
 	"github.com/urfave/cli"
 )
 
 func loadGenesis(ctx *cli.Context) (*genesis.GenesisState, error) {
+	fig, err := getConfig(ctx)
+	if err != nil {
+		log.Crit("unable to extract required config", "err", err)
```

Comment:
No need to log here, error will get returned and logged

Reply:
okay i have removed logs from this function